src/components/Header/Header.js

```diff
@@ -1,26 +1,95 @@
 import React, { useState } from 'react'
 import { FaTimes, FaBars } from 'react-icons/fa';
-import { HeaderContainer, Logo, MobileIcon, MobileMenu } from './Header.elements'
+import { IconContext } from 'react-icons/lib';
+import { 
+  HeaderContainer, 
+  HeaderWrapper, 
+  Logo, 
+  LogoIcon, 
+  MobileIcon, 
+  NavMenu,
+  NavItem, 
+  NavLinks,
+  UserOptions,
+  UserIcon
+} from './Header.elements'
 
 
 const Header = () => {
-  const [click, setClick] = useState(false)
+  const [mobileNav, setMobileNav] = useState(false)
+  const handleMobileNav = () => {
+    setUserDropdown(false) 
+    return setMobileNav(!mobileNav)
+  }
+   
+  const [userDropdown, setUserDropdown] = useState(false)
+  const handleUserDropdown = () => {
+    setMobileNav(false)
+    return setUserDropdown(!userDropdown)
+  }
 
-  const handleClick = () => setClick(!click)
+  const handleMenus = () => {
+    setUserDropdown(false) 
+    setMobileNav(false)
+  }
 
   return (
     <>
```

Comment:
Since everything is wrapped in the `<IconContext.Provider>` I'm not sure if we need these fragments to wrap the return